medicines/doc-index-updater/src/delete_manager/mod.rs

```diff
@@ -1,23 +1,27 @@
-use crate::{models::DeleteMessage, service_bus_client::delete_factory};
-use azure_sdk_core::errors::AzureError;
+use crate::{models::DeleteMessage, service_bus_client::delete_factory, storage_client};
+use azure_sdk_core::{errors::AzureError, prelude::*, DeleteSnapshotsMethod};
 use azure_sdk_service_bus::prelude::Client;
+use azure_sdk_storage_blob::prelude::*;
```

Comment:
I really dislike this import of `*`, but I can't figure out what I'm supposed to import from here. If I remove this line, `delete_blob` ceases to be a method of the Azure storage client.